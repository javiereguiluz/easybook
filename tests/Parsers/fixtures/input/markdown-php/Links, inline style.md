[silly URL w/ angle brackets](<?}]*+|&)>).

[link](<url://with spaces> "title").

[link][].

[link]: <url with spaces> "title"