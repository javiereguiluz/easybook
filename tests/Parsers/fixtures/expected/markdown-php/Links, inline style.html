<p><a href="?}]*+|&amp;)">silly URL w/ angle brackets</a>.</p>

<p><a href="url://with spaces" title="title">link</a>.</p>

<p><a href="url with spaces" title="title">link</a>.</p>
